website/src/css/_spacing.scss

```diff
@@ -52,11 +52,31 @@ $heights: 100, 200, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850;
 }
 
 // Custom max widths
-$maxWidths: 100, 200, 300, 350, 400, 450, 500, 550, 600, 650, 700;
+$maxWidths: 50, 75, 100, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700;
 
 // Custom max-width utility classes
 @each $maxWidth in $maxWidths {
     .mw-#{$maxWidth} {
         max-width: #{$maxWidth}px;
     }
+    @media (min-width: 1200px) {
+        .mw-xl-#{$maxWidth} {
+            max-width: #{$maxWidth}px;
+        }
+    }
+    @media (min-width: 769px) and (max-width: 1200px) {
+        .mw-lg-#{$maxWidth} {
+            max-width: #{$maxWidth}px;
+        }
+    }
+    @media (max-width: 768px) {
+        .mw-md-#{$maxWidth} {
+            max-width: #{$maxWidth}px;
+        }
+    }
+    @media (max-width: 576px) {
+        .mw-sm-#{$maxWidth} {
+            max-width: #{$maxWidth}px;
+        }
+    }
```

Comment:
It's beautiful :,) -- Maybe for sake of time to get this in I can do this while I do this story for the Replatform? [#68](https://github.com/sourcegraph/about-replatform/issues/68)

Reply:
Haha! Thankz! 🥲 

I think when sourcegraph/about-replatform#67 is finalized and merged, that would be helpful!

Reply:
Just a heads up that I'm gonna bring over the new Blockquotes in https://github.com/sourcegraph/about-replatform/pull/65 since I was porting `Blockquote` as part of that anyway